numpyro/infer/inspect.py

```diff
@@ -359,21 +361,31 @@ def _resolve_plate_samples(plate_samples):
     }
 
     def get_log_probs(sample):
+        class substitute_deterministic(handlers.substitute):
+            def process_message(self, msg):
+                if msg["type"] == "deterministic":
+                    msg["args"] = (msg["value"],)
+                    msg["kwargs"] = {}
+                    msg["value"] = self.data.get(msg["name"])
+                    msg["fn"] = lambda x: x
+
+
```

Comment:
It seems that CI is failing due to two blank lines. Could you run `make lint` or `make format` in the cloned numpyro folder to fix those issues

Reply:
`make lint` seems to produce an error, but `make format` works. I have pushed these changes.